sec/review.tex

```diff
@@ -246,6 +247,55 @@ \subsubsection{Centroid-based clustering}\label{subsubsec:kmeans}
 agnostic to its distance measure, and removes the requirement for the attribute
 space to be a metric space at all.
 
+\begin{table}
+    \input{tex/datasets.tex}
```

Comment:
:le_chefs:kiss:

Reply:
I was sitting there thinking "how exactly does `pandas` make these tables look so nice?" 🤔

And then it dawned on me: just make `pandas` make the table look nice 🧑‍🍳🤌